Samples/Swift/DaysUntilBirthday/Shared/Services/BirthdayLoader.swift

```diff
@@ -42,8 +42,8 @@ final class BirthdayLoader: ObservableObject {
     guard let accessToken = GIDSignIn
             .sharedInstance
             .currentUser?
-            .authentication
-            .accessToken else { return nil }
+            .accessToken
```

Comment:
Have you checked to see if the Objc sample needs similar updates?

Reply:
Done.

Reply:
Done.